daprdocs/content/en/operations/support/support-release-policy.md

```diff
@@ -0,0 +1,65 @@
+weight: 1000
+description: "Release support and upgrade policies "
+---
+
+## Introduction
+This topic details the supported versions of Dapr releases, the upgrade policies and how deprecations and breaking changes are communicated.
+
+Dapr releases use `MAJOR.MINOR.PATCH` versioning. For example 1.0.0
+
+  * A `PATCH` version is incremented for bug and security hot fixes.
+  * A `MINOR` version is updated as part of the regular release cadence, including new features, bug and security fixes. 
+  * A `MAJOR` version is updated when there’s a non-backward compatible change to the runtime, such as an API change.  A `MAJOR` release can also occur then there is a considered a significant addition/change of functionality that needs to differentiate from the previous version.
+
+A supported release means;
+
+- A hoxfix patch is released if the release has a critical issue such as a mainline broken scenario or a security issue. Each of these are reviewed on a case by case basis.
+- Issues are investigated for the supported releases. If a release is no longer supported, you need to upgrade to a newer release and determine if the issue is still relevant.
+
+From the 1.0.0 release onwards two (2) versions of Dapr are supported; the current and previous versions. Typically these are `MINOR`release updates. This means that there is a rolling window that moves forward for supported releases and it is your operational responsibility to remain up to date with these supported versions. If you have an older version of Dapr you may have to do intermediate upgrades to get to a supported version. 
+
+There will be at least 6 weeks between major.minor version releases giving users a 12 week (3 month) rolling window for upgrading.
+
+Patch support is for supported versions (current and previous).
+
+## Supported versions
+The table below shows the versions of Dapr releases that have been tested together and form a "packaged" release. Any other combinations of releases are not supported. 
+
+| Release date | Runtime     | CLI  | SDKs  | Dashboard  | Status |
+|--------------------|:--------:|:--------|---------|---------|---------|
+| Feb 17th 2021 | 1.0.0 | 1.0.0 | Java 1.0.0 </br>Go 1.0.0 </br>PHP 1.0.0 </br>Python 1.0.0 </br>.NET 1.0.0 | 0.6.0 | Supported (current) |
+
+## Upgrade paths
+After the 1.0 release of the runtime there may be situations where it is necessary to explicitly upgrade through an additional release to reach the desired target. For example an upgrade from v1.0 to v1.2 may need go pass through v1.1
+
+The table below shows the tested upgrade paths for the Dapr runtime. For example you are able to upgrade from 1.0-rc4 to the 1.0 release. Any other combinations of upgrades have not been tested.
+
+|  Current Runtime version | Must upgrade through  | Target Runtime version   |
+|--------------------------|-----------------------|------------------------- |
+| 0.11                     |               1.0-rc4 |                    1.0.0 |
```

Comment:
cc @yaron2 is this the scenario? for upgrade from 0.11 must it go through 1.0-rc.4?

Reply:
@yaron2 - How does someone go from v0.11 release to the 1.0.0 release? Do they need to go through ALL RCs or only the last one (RC4), or can they go directly. Remember we has the whole placement service upgrade in RC1, so they really need to do that first

Reply:
We need to validate this scenario is technically possible. If it isn't, the guidelines should be to uninstall 0.11 and install 1.0 fresh.

Reply:
I am sure for self hosted mode, a clean install is needed since exporters is now removed.

Reply:
Update: we can upgrade from 0.11 to 1.0.0 all the way through when using the Dapr CLI. It will not work with Helm.